.github/ISSUES/test-coverage-hanging-tests.md

```
@@ -0,0 +1,112 @@
+5. **Use Go race detector and pprof**
+
+   ```bash
+   go test -race -cpuprofile=cpu.prof -memprofile=mem.prof ./lib/...
+   go tool pprof cpu.prof
+   ```
+
+## Expected Outcome
+
+Once fixed, the workflow should be updated to include `./lib` in the coverage run. The workflow comment includes a TODO with instructions for updating:
+
+```yaml
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 4; Actual: 2

(MD007, ul-indent)